src/cartesian/getEquidistantTicks.ts

```diff
@@ -31,11 +31,19 @@ export function getEquidistantTicks(
     }
 
     // Check if the element collides with the next element
-    const size = getTickSize(entry, index);
+    const i = index;
+    let size: number | undefined;
+    const getSize = () => {
+      if (size === undefined) {
+        size = getTickSize(entry, i);
+      }
+
+      return size;
+    };
 
     const tickCoord = entry.coordinate;
     // We will always show the first tick.
-    const isShow = index === 0 || isVisible(sign, tickCoord, size, start, end);
+    const isShow = index === 0 || isVisible(sign, tickCoord, getSize, start, end);
```

Comment:
We are now redefining this in three places. I wonder: Could we instead extract this combination of getSize with isVisible into a new helper function?

Reply:
I don't think we could because there might be other places where `getSize` is used. 
For example, in `getTicksEnd` we use it in lines 37, 46 and 49. It isn't just used for `isVisible`, so it's hard to abstract that